### PR TITLE
Added Link to sumosim homepage

### PR DESCRIPTION
**Pull request by wwagner4**
Added a link to one of my page that uses scala-js

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -252,6 +252,8 @@ Want to contribute to Scala.js? Check out the
 
 List of websites using Scala.js:
 
+* Januar 2015
```
Comment (sjrd) [inline index.md]: Januar**y**

**Comment by sjrd** [comment]
Oh, could you please squash the commits, so there's only one? Fixup commits should always be squashed with the commit they're fixing. Thank you.

**Comment by sjrd** [comment]
Also, when you've updated, just drop a comment on the PR, so I get a notification. Just pushing the branch does not send me a notification, so it can be days literally before I notice there's something new ;)

**Comment by wwagner4** [comment]
Actually I am not the mastor of git, but hopefully I did it correctly now.

**Comment by sjrd** [comment]
LGTM it seems you did :)

**Pull request merged**